.github/workflows/sync_peloton_to_garmin.yml

```diff
@@ -82,3 +82,4 @@ jobs:
         P2G_PELOTON__PASSWORD: ${{ secrets.P2G_PELOTON__PASSWORD }}
         P2G_GARMIN__EMAIL: ${{ secrets.P2G_GARMIN__EMAIL }}
         P2G_GARMIN__PASSWORD: ${{ secrets.P2G_GARMIN__PASSWORD }}
+TZ: America/Los_Angeles

```

Comment:
Yaml files are very sensitive to spaces.  This environment variable needs to be tabbed over so it lines up with the other env variables

Reply:
gotcha, will go try now, thank you sooo very much!!!

Reply:
ok I now have it entered like this and its still syncing at the 8 hr's ahead
        P2G_PELOTON__PASSWORD: ${{ secrets.P2G_PELOTON__PASSWORD }}
        P2G_GARMIN__EMAIL: ${{ secrets.P2G_GARMIN__EMAIL }}
        P2G_GARMIN__PASSWORD: ${{ secrets.P2G_GARMIN__PASSWORD }}
        +TZ: America/Los_Angeles

Reply:
Ok looked back and after it not working again looks like that + was not supposed to be there, deleted and ran successfully!!!! Thanks a ton!!!!